src/main/java/se/sundsvall/chat/api/model/MessageRequest.java

```diff
@@ -0,0 +1,119 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a message within a conversation, including metadata and content.")
+public class MessageRequest {
+
+	@Schema(description = "The sequence number of the message in the conversation.", example = "1")
+	private String sequenceNumber;
```

Comment:
This is a big burden for the client to set. 

I suggest that we removes the attribute from this class (MessageRequest) and use database sequences instead.